ZipBuilder/Sources/ZipBuilder/FrameworkBuilder.swift

```diff
@@ -444,7 +493,8 @@ struct FrameworkBuilder {
     let xcframework = packageXCFramework(withName: framework,
                                          fromFolder: frameworkDir,
                                          thinArchives: thinArchives,
-                                         moduleMapContents: moduleMapContents)
+                                         moduleMapContents:
+                                         podInfo.moduleMapContents!.get(umbrellaHeader: umbrellaHeader))
```

Comment:
Are we 100% sure `moduleMapContents` exists at this point? If yes, may it be non-optional? It's better to avoid force unwrap unless it is necessary.

Reply:
Yes. And if it's not there, we should crash since we'll otherwise generate a bad framework.

Reply:
A crash from force unwrap is hardly readable. I would suggest adding a `guard` statement with a fatal error in `else` clause with an informative message.

Reply:
Fixed.